src/daemonStarter.ts

```diff
@@ -61,8 +61,8 @@ export async function tryStartDaemon(logService: ILogService, options?: Partial<
     const daemon = spawn(process.execPath, args, {
         detached: true,
         stdio: 'ignore',
-        env: process.env
+        env: { ...process.env, ELECTRON_RUN_AS_NODE: '1' },
```

Comment:
`ELECTRON_RUN_AS_NODE` what will this affect? @jeanp413

Reply:
Looks the same in the end as I saw `--enable-electron-run-as-node` was used automatically too, but I saw this used in vscode extensions so better to be explicit